docs/docs/getting-started.md

```diff
@@ -29,6 +29,10 @@ The Hello Qlik Core examples are divided into three separate parts:
     [halyard.js](https://github.com/qlik-oss/halyard.js).
 - [Hello Visualization](./tutorials/hello-visualization.md) - Building a visualization using picasso.js.
```

Comment:
Maybe use visualizationS -  "Building visualizations using picasso.js (and add link to piccaso.js at github)